packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -1,23 +1,22 @@
-import { join } from 'path';
-import * as vscode from 'vscode';
-
 /*
  * Copyright (c) 2021, salesforce.com, inc.
  * All rights reserved.
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 import {
-    channelService, ContinueResponse, getRelativeProjectPath,
-    LibraryCommandletExecutor, Row, SfdxPackageDirectories, SourceTrackingService,
-    SourceTrackingType, Table, workspaceUtils
+  channelService, ContinueResponse, getRelativeProjectPath,
```

Comment:
I think I have all of this covered in my PR - https://github.com/forcedotcom/salesforcedx-vscode/pull/4801/files.
Wen might want to revert the changes here to avoid a merge conflict.

Reply:
@jeffb-sfdc I will have a look.